docs/root/configuration/operations/overload_manager/overload_manager.rst

```diff
@@ -163,6 +167,54 @@ all listeners.
 
 An example configuration can be found in the :ref:`edge best practices document <best_practices_edge>`.
 
+.. _config_overload_manager_reset_streams:
```

Comment:
If this isn't fully implemented should we hold docs until it's usable?  otherwise please at least add a warning this only works for HTTP/2 today

Reply:
very non-intuitive. TIL.